ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -276,6 +354,13 @@ private List<String[]> readPairsFromFile(String fileLocation, String paramName)
     }
   }
 
+  private String extractReportingUrl(ObjectNode payload) {
+    if (!payload.path(Attribute.REPORTING_URL).isMissingNode()) {
+      return payload.path(Attribute.REPORTING_URL).asText();
+    }
+    return payload.path("metrics").path("url").path("top_sites_contile_reporting_url").asText();
```

Comment:
```suggestion
    return payload.path("metrics").path("url").path("top_sites.contile_reporting_url").asText();
```

The path in the JSON contains a `.` because these are constructed as `<metric_group>.<metric_name>`, but this gets normalized to an underscore for the BQ field name.